specs/binding-http.spec/src/main/resources/META-INF/zilla/http.idl

```diff
@@ -54,38 +54,19 @@ scope http
     {
         enum HttpEventType (uint8)
         {
-            AUTHORIZATION (1),
-            REQUEST (2),
-            RESPONSE (3)
+            AUTHORIZATION_FAILURE (1),
+            REQUEST (2)
         }
 
-        enum Result (uint8)
+        struct HttpDefaultEvent extends core::event::Event
```

Comment:
We need separate `HttpRequestAccepted` event and `HttpAuthorizationFailed` event.

We need the request accepted event to include `scheme`, `authority` and `path`.

Reply:
OK